field/TinyLang/Field/Generator.hs

```diff
@@ -61,9 +62,22 @@ uniVars =
     mapMaybe $ forget $ \uniVar@(UniVar uni _) ->
         withGeqUni uni (knownUni @f @a) (Just uniVar) Nothing
 
+-- | Choose a variable of a particular type.
 chooseUniVar :: (KnownUni f a, MonadGen m) => [SomeUniVar f] -> m (UniVar f a)
```

Comment:
Here and elsewhere the type [SomeUniVar f] is used, but we also have Vars f as a synonym for this.  Should we pick one of these and use it everywhere consistently?

Reply:
Yes. Fixed.